packages/terra-visually-hidden-text/docs/README.md

```diff
@@ -25,10 +25,10 @@ This component requires the following peer dependencies be installed in your app
 | react | ^16.8.5 |
 | react-dom | ^16.8.5 |
 
-
```

Comment:
Lets delete this file since the content was copy and pasted into this file: https://github.com/cerner/terra-core/pull/2882/files#diff-b3a52cab677e3c49650618c990b697bbR1

Reply:
done here https://github.com/cerner/terra-core/pull/2882/commits/91b59fdaaaa0cbeb5064a9c2469ea5801338eb6d.